### PR TITLE
DOT-4814-V2: We grouped all the gateways on a single block

### PR DESCRIPTION
**Pull request by olyans**
This PR based on https://nextmediama.atlassian.net/browse/DOT-4814

- [x] DOT-4814-V2: We grouped all the gateways on a single block
- [x] DOT-4814-V2: We need to make changes to display this order will be processed by Cash Plus

### CODE REVIEW

**Review by AmineAlyate — DISMISSED** [review]
Technical review LGTM :heavy_check_mark:

File: assets/js/youcan-pay.js [hunk 1]
```diff
@@ -38,17 +43,25 @@ jQuery(function ($) {
...
+
                 window.ycPay.pay(data.token_transaction)
                     .then(function (transactionId) {
                         detach_loader($form, loader);
 
-                        if (typeof (data.redirect_url) !== 'undefined') {
-                            window.location.href = data.redirect_url;
-                            return;
-                        }
+                        console.log("i'm here 1!");
```
Comment (AmineAlyate) [inline assets/js/youcan-pay.js:53]: remove this

File: assets/js/youcan-pay.js [hunk 2]
```diff
@@ -38,17 +43,25 @@ jQuery(function ($) {
...
+
                         if (typeof (data.redirect) !== 'undefined') {
+                            console.log('i\'m here 2!');
+                            console.log('transaction_id', transactionId);
```
Comment (AmineAlyate) [inline assets/js/youcan-pay.js:57]: those alsoe

File: assets/js/youcan-pay.js [hunk 3]
```diff
@@ -38,17 +43,25 @@ jQuery(function ($) {
...
+
                             let url = new URL(data.redirect);
                             url.searchParams.set('transaction_id', transactionId);
+                            if (gateways.cash_plus === parseInt(window.ycPay.selectedGateway)) {
+                                url.searchParams.set('gateway', 'cash_plus');
+                            }
+                            console.log('url', url.href);
```
Comment (AmineAlyate) [inline assets/js/youcan-pay.js:64]: same

File: assets/js/youcan-pay.js (hunk 2 above)
Comment (olyans) [reply assets/js/youcan-pay.js]: https://github.com/NextmediaMa/youcan-pay-for-woocommerce/pull/26/commits/e7ee34f8a6f31459dd81379f445f313c66ef2877

File: assets/js/youcan-pay.js (hunk 1 above)
Comment (olyans) [reply assets/js/youcan-pay.js]: https://github.com/NextmediaMa/youcan-pay-for-woocommerce/pull/26/commits/e7ee34f8a6f31459dd81379f445f313c66ef2877

File: assets/js/youcan-pay.js (hunk 3 above)
Comment (olyans) [reply assets/js/youcan-pay.js]: https://github.com/NextmediaMa/youcan-pay-for-woocommerce/pull/26/commits/e7ee34f8a6f31459dd81379f445f313c66ef2877

**Review by AmineAlyate — APPROVED** [review]
Technical review LGTM :heavy_check_mark:

**Pull request merged**